src/integration-test/java/com/commercetools/sync/inventories/InventoryIntegrationTestUtils.java

```diff
@@ -0,0 +1,224 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.channels.ChannelDraft;
+import io.sphere.sdk.channels.ChannelRole;
+import io.sphere.sdk.channels.commands.ChannelCreateCommand;
+import io.sphere.sdk.channels.commands.ChannelDeleteCommand;
+import io.sphere.sdk.channels.queries.ChannelQuery;
+import io.sphere.sdk.channels.queries.ChannelQueryBuilder;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+import io.sphere.sdk.inventory.InventoryEntryDraftBuilder;
+import io.sphere.sdk.inventory.commands.InventoryEntryCreateCommand;
+import io.sphere.sdk.inventory.commands.InventoryEntryDeleteCommand;
+import io.sphere.sdk.inventory.queries.InventoryEntryQuery;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.types.FieldDefinition;
+import io.sphere.sdk.types.StringFieldType;
+import io.sphere.sdk.types.TypeDraft;
+import io.sphere.sdk.types.TypeDraftBuilder;
+import io.sphere.sdk.types.commands.TypeCreateCommand;
+import io.sphere.sdk.types.commands.TypeDeleteCommand;
+import io.sphere.sdk.types.queries.TypeQuery;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Optional;
+
+import static com.commercetools.sync.commons.utils.SphereClientUtils.CTP_SOURCE_CLIENT;
+import static com.commercetools.sync.commons.utils.SphereClientUtils.CTP_TARGET_CLIENT;
+import static com.commercetools.sync.commons.utils.SphereClientUtils.QUERY_MAX_LIMIT;
+import static com.commercetools.sync.commons.utils.SphereClientUtils.fetchAndProcess;
+import static java.util.Collections.singleton;
+import static java.util.Collections.singletonList;
+
+public class InventoryIntegrationTestUtils {
```

Comment:
@jakubchrobak why not use consistent naming? sometimes you use `IT` and here you use `IntegrationTest`. Also I think this class should be in the utils package..